ocsp_status_test.go

```diff
@@ -69,7 +73,7 @@ func TestCheckFromCert(t *testing.T) {
 	mt := mocks.NewMockToolsInterface(ctrl)
 	mt.EXPECT().ParseCertificateFile(Cert).Return(&x509.Certificate{}, nil)
 	mt.EXPECT().GetIssuerCertFromLeafCert(gomock.AssignableToTypeOf(&x509.Certificate{})).Return(&x509.Certificate{}, nil)
-	// I'm not sure if there's any value actually specifying all these types
+	// I don't think there is value in actually specifying all these types
```

Comment:
i think you can omit this whole comment.

Reply:
Will do!